requirements.txt

```diff
@@ -0,0 +1,24 @@
+# The following packages are required for core functionality.
+pefile>=2017.8.1
+
+# The following packages are optional.
+# If certain packages are not necessary, place a comment (#) at the start of the line.
+
+# This is required for the yara plugins
+yara-python>=3.8.0
+
+# This is required for several plugins that perform malware analysis and disassemble code.
+# It can also improve accuracy of Windows 8 and later memory samples.
+capstone>=3.0.5
+
+# This is required by plugins that decrypt passwords, password hashes, etc.
+pycryptodome
+
+# This can improve error messages regarding improperly configured ISF files.
+jsonschema>=2.3.0
+
+# This is required for memory acquisition via leech.
```

Comment:
I think it's either leechcore or pcileech.  Just leech on its own isn't quite enough to help people track it down...